src/pages/projectSettings/tabs/NotificationsTab/getFormSchema.tsx

```diff
@@ -12,53 +16,86 @@ export const getFormSchema = (
-            oneOf: radioBoxOptions(
-              ["Enabled", "Disabled"],
-              repoData?.buildBreakSettings?.notifyOnBuildFailure
-            ),
+} => {
+  const { schema: eventSchema, uiSchema: eventUiSchema } = getEventSchema(
+    [],
+    projectTriggers
+  );
+  const {
+    schema: notificationSchema,
+    uiSchema: notificationUiSchema,
+  } = getNotificationSchema(subscriptionMethods);
+
+  return {
+    fields: {
+      subscriptionField: SubscriptionField,
+    },
+    schema: {
+      type: "object" as "object",
+      properties: {
+        buildBreakSettings: {
+          type: "object" as "object",
+          title: "Performance Plugins",
+          properties: {
+            notifyOnBuildFailure: {
+              type: ["boolean", "null"],
+              title: "Build Break Notifications",
+              oneOf: radioBoxOptions(
+                ["Enabled", "Disabled"],
+                repoData?.buildBreakSettings?.notifyOnBuildFailure
+              ),
+            },
           },
         },
-      },
-      subscriptions: {
-        type: "array" as "array",
-        title: "Subscriptions",
-        items: {
-          type: "object" as "object",
+        subscriptions: {
+          type: "array" as "array",
+          title: "Subscriptions",
+          items: {
+            type: "object" as "object",
+            properties: {
+              subscriptionData: {
+                type: "object" as "object",
+                title: "",
+                properties: {
+                  event: eventSchema,
+                  notification: notificationSchema,
+                },
+              },
+              subscriberData: {
+                type: "object" as "object",
+                properties: {},
+              },
+            },
+          },
         },
       },
     },
-  },
-  uiSchema: {
-    buildBreakSettings: {
-      "ui:rootFieldId": "plugins",
-      "ui:ObjectFieldTemplate": CardFieldTemplate,
-      notifyOnBuildFailure: {
-        "ui:widget": widgets.RadioBoxWidget,
-        "ui:description":
-          "Send notification of build breaks to admins of a project if the commit author is not signed up to receive notifications.",
+    uiSchema: {
+      buildBreakSettings: {
+        "ui:rootFieldId": "plugins",
+        "ui:ObjectFieldTemplate": CardFieldTemplate,
+        notifyOnBuildFailure: {
+          "ui:widget": widgets.RadioBoxWidget,
+          "ui:description":
+            "Send notification of build breaks to admins of a project if the commit author is not signed up to receive notifications.",
+        },
       },
-    },
-    subscriptions: {
-      "ui:placeholder": "No subscription are defined.",
-      "ui:addButtonText": "Add Subscription",
-      "ui:orderable": false,
-      items: {
-        "ui:field": "subscriptionField",
+      subscriptions: {
+        "ui:placeholder": "No subscription are defined.",
+        "ui:addButtonText": "Add Subscription",
```

Comment:
The title no longer shows up
<img width="733" alt="image" src="https://user-images.githubusercontent.com/13104717/175551281-9f3b8955-ecfe-4cb4-b0c8-c27493c46726.png">

how it used to be: 
<img width="533" alt="image" src="https://user-images.githubusercontent.com/13104717/175551491-adf1783c-ca80-4c10-993d-5fdf1fcd66ab.png">

do we want to add it back in now, or do it in a separate ticket?